src/Database/DatabaseManager.cs

```diff
@@ -63,16 +63,16 @@ public static IHealthChecksBuilder AddDatabaseHealthCheck(this IHealthChecksBuil
             }
         }
 
-        public static IServiceCollection ConfigureDatabase(this IServiceCollection services, IConfigurationSection? connectionStringConfigurationSection, ILogger logger)
-            => services.ConfigureDatabase(connectionStringConfigurationSection, new FileSystem(), logger);
+        public static IServiceCollection ConfigureDatabase(this IServiceCollection services, IConfigurationSection? connectionStringConfigurationSection, IConfigurationSection? pluginsConfigurationSection, ILogger logger)
```

Comment:
Is `pluginsConfigurationSection` used?

Reply:
Not ATM but will be for a plugin were writing and would be the way to pass plugin config for future plugins